### DISCUSSION TITLE
Hosting

### DISCUSSION THREAD

**Discussion by Jorgelzn** [samcf/ogres#145] [Q&A] [answered]
Hi !

First of all, awesome project !!

I wanted to ask you if you are hosting this app on github pages so in case i want to deploy it I can also use github pages. In case I need a server, could I host it locally and have friends remotely connecting by browser ?

Kind regards

**Comment by samcf** [comment, marked as answer]
Hi @Jorgelzn, its not hosted on Github pages anymore, now I use a private server from Digital Ocean. You'll have to find something that can host a Docker image - GIthub pages only serves static content which isn't enough to host the app.

As always you can use https://ogres.app/ for free!

**Comment by Jorgelzn** [comment]
Hi samcf, thanks for answering.

Ok, I will check out ogress.app then :D
